clusterloader2/api/types.go

```diff
@@ -142,15 +142,28 @@ type TuningSet struct {
 	GlobalQPSLoad *GlobalQPSLoad `json: globalQPSLoad`
 }
 
+// MeasurementConfig is a structure that contains the Identifier of a singular measurement OR identifier + params for a wrapper measurement.
+// Only one of the 2 should be supplied and supplying both would result in an error.
+type MeasurementConfig struct {
+	// Identifier is an interface{} which could be a string or map of {name: value} that differentiates measurement instances of the same method.
+	Identifier string `json: identifier`
+	// Params is a map of {name: value} pairs which will be passed to the measurement method - allowing for injection of arbitrary parameters to it.
+	Params map[string]interface{} `json: params`
+}
+
 // Measurement is a structure that defines the measurement method call.
 // This method call will either start or stop process of collecting specific data samples.
 type Measurement struct {
 	// Method is a name of a method registered in the ClusterLoader factory.
 	Method string `json: method`
-	// Identifier is a string that differentiates measurement instances of the same method.
-	Identifier string `json: identifier`
 	// Params is a map of {name: value} pairs which will be passed to the measurement method - allowing for injection of arbitrary parameters to it.
 	Params map[string]interface{} `json: params`
+	// Exactly one of Identifier and Instances must be supplied. Identifier is for single measurements while Instances
```

Comment:
nitt add an empty line before it to make it explicit that those are connected. 
Also, make it explicit that it's exclusive with the other in each field comment.
```
// Identifier is a string that differentiates measurement instances of the same method.
// Identifier can't be set together with Instances field.
Identifier string `json: identifier`
// Instances contains a list of instances of this measurement.
// Instances can't be set together with Identifier field.
Instances []MeasurementInstance `json: instances`